flowzone.yml

```diff
@@ -1,10 +1,26 @@
 .flowzone:
 
+  - &checkSecrets
+    name: Check secrets
+    id: check_secrets
+    run: |
+      if [[ -z '${{ secrets.FLOWZONE_TOKEN }}' ]] \
+        && [[ -z '${{ secrets.GH_APP_PRIVATE_KEY }}' ]]; then
+          echo '::error::Must specify either GH_APP_PRIVATE_KEY or FLOWZONE_TOKEN.'
+          false
+      fi
+
+      if [[ -n '${{ secrets.GH_APP_PRIVATE_KEY }}' ]]; then
+          echo 'private_key=true' >> $GITHUB_OUTPUT
+      else
+          echo 'private_key=false' >> $GITHUB_OUTPUT
+      fi
+
   - &getEphemeralToken
     # https://github.com/marketplace/actions/github-app-token
     name: Generate token
     uses: tibdex/github-app-token@v1.8.0
-    if: needs.context_check.outputs.ephemeral_tokens
```

Comment:
missed this regression after a rebase with Flowzone reverting silently to FLOWZONE_TOKEN.